src/app/main.cpp

```diff
@@ -445,8 +450,14 @@ int main(int argc, char **argv)
 #ifdef Q_OS_WIN
     else {
         // set the windows settings userdir to the install dir
-        QDir rootDir = QApplication::applicationDirPath();
+        // do not use QApplication::applicationDirPath() befor create QApplication, becouse
+        // if installer started from other drive (D:\, E:\ etc.) it return the launch dir, not the app dir
```

Comment:
I think it would be better to not write about the installer here - it should be sufficient to say that it returns the launch dir instead of app dir.